src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java

```diff
@@ -186,6 +188,7 @@ public void testCreateUser_withProgramTokenInUserObject() throws Exception {
         assertThat(apiClientUser, is(notNullValue()));
         assertThat(apiClientUser.getFirstName(), is(equalTo("test-first-name")));
         assertThat(apiClientUser.getStatus(), is(nullValue()));
+        assertThat(apiClientUser.getVerificationStatus(), is(nullValue()));
```

Comment:
Please add a test case that validates that `verificationStatus` is not a null value and with any of the following values `enum VerificationStatus {EXPIRED, FAILED, UNDER_REVIEW, VERIFIED, REQUIRED, NOT_REQUIRED}`